lib/multi_blas_quda.cu

```diff
@@ -557,7 +571,7 @@ namespace quda {
     {
       std::vector<double> a_(x.size() * y.size());
       memcpy(a_.data(), a, x.size() * y.size() * sizeof(double));
-      std::vector<std::reference_wrapper<ColorSpinorField>> x_;
+      std::vector<std::reference_wrapper<const ColorSpinorField>> x_;
```

Comment:
Here and later in this file, why not use `vector_ref`?

Reply:
[b607f03](https://github.com/lattice/quda/pull/1318/commits/b607f03e157ea8c08aae26a419c4c56ad5d6f40a)